tests/test_bias.py

```diff
@@ -0,0 +1,182 @@
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import hashlib
+import os
+import numpy as np
+import unittest
+
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+import lsst.ip.isr as ipIsr
+import lsst.meas.algorithms as measAlg
+import lsst.utils.tests
+
+from lsst.pipe.tasks.repair import RepairTask
+from lsst.utils import getPackageDir
+
+
+class BiasTestCases(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        """Setup butler, and generate an ISR processed exposure.
+
+        Notes
+        -----
+        DMTN-101 4.1:
+
+        Process an independent bias frame through the ISR including
+        overscan correction and bias subtraction
+
+        """
+        repoDir = os.path.join(getPackageDir('ci_cpp_gen2'), "DATA")
+        calibDir = os.path.join(getPackageDir('ci_cpp_gen2'), "DATA", "calibs")
+        butler = dafPersist.Butler(repoDir, calibRoot=calibDir)
+
+        self.config = ipIsr.IsrTaskConfig()
+        self.config.doSaturation = True
+        self.config.doSuspect = True
+        self.config.doSetBadRegions = True
+        self.config.doOverscan = True
+        self.config.doBias = True
+        self.config.doVariance = True
+
+        self.config.doLinearize = False
+        self.config.doCrosstalk = False
+        self.config.doWidenSaturationTrails = False
+        self.config.doBrighterFatter = False
+        self.config.doDefect = False
+        self.config.doSaturationInterpolation = False
+        self.config.doDark = False
+        self.config.doStrayLight = False
+        self.config.doFlat = False
+        self.config.doApplyGains = False
+        self.config.doFringe = False
+        self.config.doMeasureBackground = False
+        self.config.doVignette = False
+        self.config.doAttachTransmissionCurve = False
+        self.config.doUseOpticsTransmission = False
+        self.config.doUseFilterTransmission = False
+        self.config.doUseSensorTransmission = False
+        self.config.doUseAtmosphereTransmission = False
+
+        self.isrTask = ipIsr.IsrTask(config=self.config)
+        # This is not an independent frame.
+        self.dataRef = butler.dataRef('raw', dataId={'detector': 0, 'expId': 2020012800007})
+        results = self.isrTask.runDataRef(self.dataRef)
+        self.exposure = results.outputExposure
+
+    def test_canary(self):
```

Comment:
Why is this called "canary"?

Reply:
And why do you need to check for value changes?

Reply:
I originally put this in to detect algorithm changes, but I suspect this will end up being overly restrictive.  Removed throughout.